lib/pages/settings_screen.dart

```diff
@@ -47,9 +50,93 @@ class _SettingsScreenState extends State<SettingsScreen> {
           },
         ),
       ),
-      body: const Center(
-        child: Text('Settings'),
-    ),
+      body: Stack(
+        children: [
+          Container(
+            padding: const EdgeInsets.all(16),
+            color: const Color(0xFF1A1A1A), // Set background color to black
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Row(
+                  children: [
+                    SizedBox(
+                      width: 100,//size of profile image
+                      height: 100,//size of profile image
+                      child: CachedNetworkImage(
```

Comment:
Perhaps change the profile picture to a circular shape using BorderRadius.circular